### ISSUE TITLE
Add a Dekorate disclaimer to all generated files

### ISSUE THREAD

**Issue by pjgg** [dekorateio/dekorate#1015]
Some Quarkus extensions as [Quarkus-helm](https://github.com/quarkiverse/quarkus-helm) are using internally Dekorate library in order to generate some k8s yamls from a Quarkus `application.properties`. From a user experience, I think that would be great to add a disclaimer on top of these files to keep inform the developers that this files were autogenerated. 

For example: 

```
 Autogenerated by Dekorate

 DO NOT EDIT DIRECTLY

```

**Comment by rsvoboda** [comment]
@Sgitario fyi

**Comment by Sgitario** [comment]
cc @iocanel @aureamunoz

**Comment by iocanel** [comment]
I agree with the idea. I am not sure about the technical challenges as we depend on jackson.
